addons/resource.language.en_gb/resources/strings.po

```diff
@@ -13448,18 +13447,10 @@ msgstr ""
 
 #: Unknown
 msgctxt "#33074"
-msgid "This version of %s requires an"
-msgstr ""
-
-#: Unknown
-msgctxt "#33075"
-msgid "Kodi revision of %s or greater to run."
+msgid "This version of %s requires a Kodi revision of %s or greater to run. Please update Kodi."
```

Comment:
As dicussed on IRC.. I think we can remove that one as it's not used within core.